README.md

```diff
@@ -79,6 +79,27 @@ jobs:
           dispatch_event_type: my-project-release
 ```
 
+If your event has **more than 10** top-level properties, you can embed it in a JSON string `{"event_payload":"<payload_as_json_string>"}` to overcome Github api limits.
```

Comment:
Today I learned about the 10 top level limit! Is this documented somewhere?

Reply:
Sure, here's the link https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#create-a-repository-dispatch-event